nbs/sidebar.yml

```diff
@@ -2,3 +2,4 @@ website:
   sidebar:
```

Comment:
_Commented on notebook `nbs/sidebar.yml` Cell 1 Line 2_
```python
website:
  sidebar:
```
test

_Respond and view the context [here](http://localhost:3000/GitNotebooks/docs/pull/2)._

Reply:
a reply